.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,92 @@
+---
+description: Report a bug.
+labels: [bug]
+name: Bug
+title: "<SCOPE>: <SUMMARY>"
+
+body:
+  - type: "checkboxes"
+
+    attributes:
+      label: I assert that this issue is relevant for Stylix
+      description: >-
+        If you need help or are unsure whether this is a Stylix bug, please
+        consider reading the [Documentation](https://stylix.danth.me) or asking
+        for help in a [GitHub Discussion](
+        https://github.com/danth/stylix/discussions) or the [Stylix Matrix
+        Space](https://matrix.to/#/#stylix:danth.me) before opening an issue.
+      options:
+        - label: >-
+            I assert that this is a bug and not a user error or support
+            request.
+          required: true
+        - label: >-
+            I assert that this is not a duplicate of an [existing
+            issue](https://github.com/danth/stylix/labels/bug).
```

Since we sometimes forget to manually add the "bug" label, this URL does not list all bugs. Should we instead just link to the issue tab:

```suggestion
            issue](https://github.com/danth/stylix/issues).
```

---

i thought about that, but I think that we should keep it as each template adds its respective label automatically, so this will not be a problem in the future.

though, we could have it be general for now and some time in the future (once a higher proportion of bugs are properly labeled) change it to be more specific.

---

> i thought about that, but I think that we should keep it as each template adds its respective label automatically, so this will not be a problem in the future.

True.

> though, we could have it be general for now and some time in the future (once a higher proportion of bugs are properly labeled) change it to be more specific. thoughts?

Sounds also good, although we are somewhat likely to forget this in the future, say in 6 months. For now, it might be better to keep the URL general and add the more specific one in the future.